src/components/Card/ProductCard/ProductCard.css

```diff
@@ -0,0 +1,9 @@
+.card {
+  width: 19rem;
+  height: max-content;
+  margin: 1rem;
+  background-color: var(--off-white);
+}
+.card-img:hover {
+  cursor: pointer;
+}
```

Comment:
if this is specific to product cards only, then its better to use CSS modules for it

Reply:
Yeah added CSS modules for that